app/src/components/items/viewer/viewer.tsx

```diff
@@ -10,14 +10,56 @@ interface ItemProps {
   canvasIndex: number;
 }
 
+const uv_config = {
```

Comment:
again, as mentioned above, not sure if this takes priority over the config in the useEffect()